frontend/src/app/libs/container/store/store.module.ts

```diff
@@ -1,15 +1,10 @@
-import { sampleFeatureKey, reducer } from './sample/sample.reducer';
-import { SampleEffects } from './sample/sample.effects';
-import { EffectsModule } from '@ngrx/effects';
 import { NgModule } from '@angular/core';
 import { StoreModule } from '@ngrx/store';
+import { EffectsModule } from '@ngrx/effects';
+import { sessionFeatureKey, sessionStateReducer } from './session/session.reducer';
+import { SessionEffects } from './session/session.effects';
 
 @NgModule({
-	imports: [
-		EffectsModule.forFeature([SampleEffects]),
-		StoreModule.forFeature(sampleFeatureKey, {
-			sample: reducer
-		})
-	]
+	imports: [EffectsModule.forFeature([SessionEffects]), StoreModule.forFeature(sessionFeatureKey, sessionStateReducer)],
```

Comment:
each store must be separate module, which inits in parent module so better revert this change and move session store init into sign-in.module.ts